### PR TITLE
docs: update README file

### LINKED ISSUE #1: docs: Initialize README.md file.
## Description
As a contributor,
I need to learn about the project,
so that I can collaborate with others on its development.

## Mocks
```
# bridge-in-tech-android


## Contributing

**This project is under active development**

Please read our [Contributing Guidelines](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/REPORTING_GUIDELINES.md) thoroughly.

## Branches

- **master**: This branches contain the deployment of the backend.
- **develop**: This contains the latest code. All the contributing PRs must be sent to this branch.

## Contact

If you have any questions or want to discuss something about this repo, feel free to join our [Zulip Community](http://anitab-org.zulipchat.com/). If you are a new contributor, head over to this project's stream [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) on Zulip to see ongoing discussions.

## License

The project is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](https://github.com/anitab-org/anitab-org.github.io/blob/develop/LICENSE) file.
```

### Update [Required]
- [ ] Include the minimum details that were included in the **Mocks** section above.
### Enhancement to Update [Optional]
- [ ] Reach out to @mtreacy002 for a description of the project!

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by menina763**
### Description

Made suggested changes to README file.

Fixes #1 and replaced PR#2

### Type of Change:

- Documentation

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by mtreacy002** [comment]
Perfect... Congratulations, @Menina294 ... You have managed to fix your develop branch 🎉 🥳 💃

**Comment by menina763** [comment]
Woo🎉! Glad that my develop branch is working now!

**Comment by mtreacy002** [comment]
You're awesome, @Menina294. Thank you for the resilience and commitment you've shown on your first contribution to AnitaB open source. 💖 . I'm honoured that you chose to work on this repo as your first step to open source. 💕

**Comment by mtreacy002** [comment]
I'll soon give my review on this PR. I'll let you know when I'm done 😉 .

**Comment by menina763** [comment]
Aww, thanks so much for helping me along the way @mtreacy002 and @isabelcosta 💖! I always felt intimidated by reaching out (especially in the backend repos), but I'm really glad that you two were able to help me along the way 😊. I'll absolutely try my best to contribute to bridge-in-tech backend and maybe learn a thing or two along the way.

Also I hope you two don't have to deal with too many gramatic errors because I'm prone to those 😅😂.

**Comment by mtreacy002** [comment]
> Aww, thanks so much for helping me along the way @mtreacy002 and @isabelcosta 💖! I always felt intimidated by reaching out (especially in the backend repos), but I'm really glad that you two were able to help me along the way 😊. I'll absolutely try my best to contribute to bridge-in-tech backend and maybe learn a thing or two along the way.
> 
> Also I hope you two don't have to deal with too many gramatic errors because I'm prone to those 😅😂.

You're most welcomed, @Menina294. Look forward to work together with you in our journey ahead 😉. Don't worry about grammar. I'm a bit slack in that area (rarely double check what I've written and just post comments in a rush and quite often too late to realise that I made grammar errors). 😂 😊  ... just made an update here even... you see what I mean...😂

File: .github/README.md
Comment (mtreacy002) [inline .github/README.md]: I think you forgot this part where @isabelcosta [requested in the previous PR](https://github.com/anitab-org/bridge-in-tech-backend/pull/2#discussion_r421863271) this description of the project (line 8 & 10) should be placed directly under the page title (line1). So can you move it to line 3, @Menina294?

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
I just requested a little change here. Other than that, it looks find to me, @Menina294.

File: .github/README.md
Comment (menina763) [reply .github/README.md]: Shoot, I thought I added that one! I'll make sure to include it.

**Comment by menina763** [comment]
Just to make sure, once I commited it, I click on "Publish Branch" and then I click on "origin" right?

**Comment by mtreacy002** [comment]
> Just to make sure, once I commited it, I click on "Publish Branch" and then I click on "origin" right?

You meant `push too...` then choose `origin`? Yes, that's right.

Screen Shot 2020-05-11 at 12 09 52 pm
Screen Shot 2020-05-11 at 12 10 00 pm

**Comment by menina763** [comment]
Okay, just pushed it! Now what do I do next? I just want to double check.

**Review by mtreacy002 — DISMISSED** [review]
Well done, @Menina294. Now we just to wait for another reviewer to approve this change as well. I've already requested @isabelcosta to give her review. Thanks again for your contribution.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@Menina294 this PR is almost done :)
There is just one little change left, your README is inside `.github` folder, when it should be in the root folder of this repository (the folder that contains the `.github` folder). Can you move it to the root folder of the repository?

**Comment by menina763** [comment]
Sorry for not responding sooner! I've been having a lot of internet and notification issues from my side. It's pretty late for me right now so I will try to make @isabelcosta 's suggested changes tommorow. Again, sorry!

**Comment by menina763** [comment]
So to move the README file @isabelcosta, do I simply have to go into my folders and move it there? Or is there an easier way in VSCode?

**Comment by isabelcosta** [comment]
@Menina294 you can simply move the file on folders or drag and drop on VScode :) you are almost done with this 👏 💪 just move it, commit and push and I am more than happy to approve this 😄

**Comment by menina763** [comment]
@isabelcosta So I have a feeling I may have messed it up again. I believe that I pushed it first and then commited it. Should I push it again?

Also, I moved the file by using ctrl X and ctrl V, but now it shows two README files. Should I simply commit it again and get rid of the file in the .github folder?

image
image

**Comment by isabelcosta** [comment]
@Menina294 as per your images, the README appears to be already in the right place :) so now you have to commit and push the change :D

**Comment by menina763** [comment]
I believe I pushed it, but just to double check, I was to push to my `origin` branch or my `upstream` branch?

**Comment by isabelcosta** [comment]
@Menina294 you can push to `origin` :) that will update automatically this PR

**Comment by meenakshi-dhanani** [comment]
@isabelcosta I see the README has been moved outside .github. Could you re-review?

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@Menina294 you are almost done ;) Can you remove/delete the extra README you have under `.github`? You have 2 READMEs at the moment, one in the right place (root folder) and another in `.github` folder.

**Comment by menina763** [comment]
Sorry for the late response @isabelcosta! I was unsure at first as to if it was okay to delate it because I used cut/copy to make the README outside that github folder and for soem reason it still stayed in there.

**Review by isabelcosta — APPROVED** [review]
I think it's all good now :) Great work @menina294! Thank you for your contribution👏🏾🎉

**Comment by menina763** [comment]
Oh my lord thanks @isabelcosta for helping me out with this, even though this took a lot longer that it should have 😅, and thanks to @mtreacy002 for helping me figure out the basics of GitHub. 🎉

**Comment by isabelcosta** [comment]
@Menina294 you are welcome :D You did amazing! You were very very patient which is critical when doing open source contributions. I am super happy we were part of your learning journey 🤗

**Review by mtreacy002 — APPROVED** [review]
This looks good, @Menina294. Thanks soo much for your contribution. 🎉

**Comment by meenakshi-dhanani** [comment]
Really appreciate your efforts @Menina294 ! Thank you :)

**Pull request merged**
